rest-api/src/main/resources/definitions/paged_resources/activity_event.yml

```diff
@@ -0,0 +1,14 @@
+type: object
+readOnly: true
+properties:
+    items:
+        type: array
+        items:
+            $ref: ../activity_event.yml
+    total:
+        type: integer
+        description: The total number of records in the items list
+    type:
+        type: string
+        readOnly: true
+        description: "ResourceList"
```

Comment:
This is all correct... but the BridgePF code is returning something different. See comments in that project.